src/styles/topbar.styl

```diff
@@ -1,6 +1,6 @@
 @require 'cozy-ui'
 
-$coz-bar-size = 40px
+$coz-bar-size = 48px
```

Comment:
Yeah i'm using it elsewhere, I replaced by 2.5em just two of them cause it didn't fit anymore.